docs/CONTRIBUTING.md

```diff
@@ -93,22 +93,12 @@ The reviewers are also responsible to check:
 
 All Pull Requests must contain proper title & description.
 
-Some Pull Requests can be exempt of `prdoc` documentation, those
-must be labelled with
+Some Pull Requests can be exempt of `prdoc` documentation, those must be labelled with
 [`R0-silent`](https://github.com/paritytech/labels/blob/main/ruled_labels/specs_polkadot-sdk.yaml#L89-L91).
 
 Non "silent" PRs must come with documentation in the form of a `.prdoc` file.
```

Comment:
Is this being enforced? I haven't been prompted by CI to write a prdoc yet, but have been really excited  for it.

Reply:
The check currently runs **but** the outcome is ignored. So today already you can see in the CI logs whether your PRdoc looks OK or not.
It will be enforced once this PR is merged.